buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -11,4 +11,22 @@ object Versions {
     const val mockk = "1.10.0"
 
     const val randomBeans = "3.9.0"
+
+    object Android {
+        const val gradlePlugin = "4.0.0"
+
+        const val compileSdk = 28
+        const val targetSdk = 28
+        const val minSdk = 21
```

Comment:
Can we decrease the version? Some of our users have lower version of SDK and by increasing the version we lose that kind of users. Allure-android has 15th minSdk version.

Reply:
Yup, of course. It is on my TODO list, I forgot it was included in this PR. We can also increase the target sdk to API 29. There were some changes to the external storage so we might have to change a couple of things. 

I am nearly finished with the migration, but it works on the versions specified in here. I will go through the codebase later on and add backwards compatibility if needed.